README.md

```diff
@@ -11,6 +11,7 @@ Kilo is a multi-cloud network overlay built on WireGuard and designed for Kubern
 ## Overview
 
 Kilo connects nodes in a cluster by providing an encrypted layer 3 network that can span across data centers and public clouds.
+Kilo enables full connectivity of the pod network, even if the cluster spans across different networks, including networks behind NAT.
```

Comment:
Somehow this sounds repetitive sine nodes in different locations (from the line below) are expected to be in different networks. Maybe we can combine this with the following line?

```suggestion
By allowing pools of nodes in different locations and networks, including networks behind NAT, to communicate securely, Kilo enables the operation of multi-cloud clusters.
```

?

Reply:
i dunno... I want to make this clear without being too repetitive and having all the sentences say "Kilo enables ..."